src/main/java/hw_17/Test2.java

```diff
@@ -0,0 +1,40 @@
+package hw_17;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.FluentWait;
+import org.openqa.selenium.support.ui.Wait;
+import java.time.Duration;
+import org.junit.Assert;
+import java.util.function.Function;
+
+public class Test2 {
+    public static void main(String[] args) {
+        WebDriver driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.get("https://makeup.com.ua/ua/");
+
+        Wait<WebDriver> fluentWait = new FluentWait<>(driver)
+                .withTimeout(Duration.ofSeconds(15))
+                .pollingEvery(Duration.ofMillis(500))
+                .ignoring(NoSuchElementException.class);
+
+        WebElement promotionsButton = fluentWait.until(new Function<WebDriver, WebElement>() {
+            public WebElement apply(WebDriver driver) {
+                return driver.findElement(By.cssSelector("a[href='/ua/actions/']"));
+            }
+        });
+        promotionsButton.click();
+
+        WebElement promotionsHeader = fluentWait.until(new Function<WebDriver, WebElement>() {
+            public WebElement apply(WebDriver driver) {
+                return driver.findElement(By.cssSelector("a[href='/ua/actions/']"));
```

Comment:
a[href='/ua/actions/'] - this element isn't a promotionsHeader and it isn't identical. Please fix it to be an identical element
![Screenshot_6](https://github.com/user-attachments/assets/d6e3b644-0beb-4452-a53c-5e42c6dd1249)

Reply:
done